feature_engine/discretisation/arbitrary.py

```diff
@@ -39,6 +40,12 @@ class ArbitraryDiscretiser(BaseNumericalTransformer):
         Whether the output, that is the bins, should be the interval boundaries. If
         True, it returns the interval boundaries. If False, it returns integers.
 
+    errors: string, default='ignore'
+        Indicates what to do if no value is assigned to one or more intervals in a
```

Comment:
We need to re-word this docstring. This is not correct. @Morgan-Sell can you change it to:

Indicates what to do when a value is outside the limits indicated in the `binning_dict`. If 'raise', the transformation will raise an error. If 'ignore', values outside the limits are returned as NaN and a warning will be raised instead.